demo-us-6.2.md

```
@@ -0,0 +1,91 @@
+# Demo: US-6.2 — Cover Mode
+
+**Story**: As a musician, I want to create a cover version of a clip in a different style.
+
+**Command**: `acemusic cover <clip_id> --style "jazz piano trio" [--lyrics "..."] [--voice <id>]`
+
+## CLI Surface
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)